pkgs/by-name/th/thcrap-steam-proton-wrapper/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  bash,
+  subversion,
+  gnome,
+}:
+with lib; let
+  version = "2b636c3";
```

```suggestion
  version = "0-unstable-2024-04-03";
```

See the [versioning conventions](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming)